changelog/dev-woopmnt-5242-e2e-tests-all-wc-770-wcpay-shopper-failing

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: dev
+
+Fix E2E test failured due to Bancontact payment method selection during checkout
```

Comment:
E2E tests are not distributed with the .org plugin, and should not be included as changelog entries.

With some log types in `npm run changelog`, you can leave the changelog entry empty, and add a comment instead. I think it would be more suitable to do so.